src/webui/CMakeLists.txt

```diff
@@ -10,7 +10,6 @@ api/synccontroller.h
 api/torrentscontroller.h
 api/transfercontroller.h
 api/serialize/serialize_torrent.h
-extra_translations.h
```

Comment:
you should also remove this file from repo?

Reply:
Did I forget to do that? Hell, it's a cost of development "on the knee".